x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/EsqlTestUtils.java

```diff
@@ -1060,6 +1060,13 @@ private static ExponentialHistogram randomExponentialHistogram() {
             ExponentialHistogramCircuitBreaker.noop(),
             rawValues
         );
+        /*
+         * hashcode mutates the histogram by doing some lazy work.
```

Comment:
Here I mean it makes something lazily. I don't mean that we were lazy when we wrote it. It's a lot of code....